docs/tech-reference/rsync-distributor.rst

```diff
@@ -0,0 +1,105 @@
+=====================
+RPM rsync Distributor
+=====================
+
+Purpose:
+========
+The RPM rsync distributor publishes RPM content to a remote server. As the name suggests, the
+distributor uses rsync to copy.
+
+Configuration
+=============
+Here's example of rpm_rsync_distributor configuration:
+
+.. code-block:: json
+
+    {
+     "distributor_id": "rpm_rsync_distributor",
+     "distributor_type_id": "rpm_rsync_distributor",
+     "distributor_config": {
+        "remote": {
+            "auth_type": "publickey",
+            "ssh_login": "foo",
+            "key_path": "/home/user/.ssh/id_rsa",
+            "host": "192.168.121.1",
+            "root": "/home/foo/pulp_root_dir"
+        },
+        "predistributor_id": "yum_distributor",
+        "force_full": true
+        }
+    }
+
+``auth_type``
+  Two authentication methods are supported: ``publickey`` and ``password``.
+
+``ssh_login``
+  ssh login to remote machine
+
+``key_path``
+  A path to the private key that will be used as identity file for ssh. When ``auth_type`` is
+  'publickey' this is a required config. The key has to be owned by user and group `apache`.
+
+``ssh_password``
+  Password is required when ``auth_type`` is 'password'.
+
+``host``
+  remote hostname
+
+``root``
+  Remote root directory with all the data (content and published contnet). Remote equivalent to
```

Comment:
I don't understand this option. Does this option let you choose where data will be placed on the remote system? Is it an absolute path or a relative path?